etc/docker/kubernetes/README.rst

```diff
@@ -0,0 +1,63 @@
+========================================
+Setting up Rucio in a Kubernetes cluster
+========================================
+
+
+Ingress configuration
+---------------------
+Depending on the platform hosting the Kubernetes cluster a variety of ingress controllers are available. The cluster currently used only supports Traefik so this readme only describes this but any other ingress controllers should work as well.
+
+First, to be able to serve https request a valid certificate/key is needed. Those have to be added to the cluster as a secret::
+
+    $ kubectl create secret tls tls-secret --key tls.key --cert tls.crt --namespace=kube-system
+
+Next, the provided traefic.toml configuration file has to be added to the cluster::
+
+    $ kubectl create configmap logging-configmap --from-file=ingress/traefik.toml --namespace=kube-system
+
+Then the daemonset configuration can be applied to the cluster::
+
+    $ kubectl apply -f ingress/traefik-ds.yaml
+
+This will create the traefik daemonset and mount the certificate and configuration file. The daemonset works with a node selector, i.e., the ingress controller will only be run on nodes that are labeled. You can label a node with::
+
+    $ kubectl label node <node-name> role=ingress
+
+After that the ingress controller should be up and running. You can check by trying to access the dashboard at: http://<node-name>:8080/dashboard/
+
+
+Starting a server instance
+--------------------------
+
+A server deployment is available under server/deployment.yaml. This will start up two pods running Rucio without TLS, expose port 80 as a service and add a new rule to the ingress with TLS termination.
+But before applying this deployment a valid rucio.cfg has to be added to the cluster::
+
+    $ kubectl create secret generic server-cfg --from-file=rucio.cfg
```

Comment:
rucio.cfg is missing in the repo. Would be good to add a reference one or the info about it

Reply:
the reference one is rucio.cfg.template in etc/